.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Committing this is fine, but for learning's sake I'll mention that by committing these settings you'd be forcing other contributors to use the same settings

Reply:
good to know! ill avoid committing this in the future 👍